example/src/shared/Wysiwyg/utils/utils.js

```diff
@@ -170,5 +171,20 @@ export const titleHandler = (editor, titleType) => {
     { line: currentLine, ch: lineContent.length }
   );
   editor.current.replaceSelection(textToInsert);
-  setTimeout(() => editor.current.focus(), [0]);
+  setTimeout(() => editor.current.focus(), 0);
+}
+
+export const insertImage = (editor, files) => {
+  let {line} = editor.current.getCursor();
+
+  files.forEach((file, i) => {
+    //create a new line after first image markdown inserted
+    if(i > 0) {
```

Comment:
Here we might need to handle the case when the cursor is not at the beginning of the line for the first file to add:

```
if (i > 0 || (i === 0 && ch !== 0)) {
 // ...
}
```